docs/fargate.md

```diff
@@ -1,6 +1,8 @@
 # Fargate
 
-**NOTE: This is an experimental feature and it's not a subject of breaking changes policy.**
+| Status    |         |
+| --------- | ------- |
+| Stability | [alpha] |
```

Comment:
Why do we use Open Telemetry stability here?

I believe Fargate is more complex in terms of support and it is not fully dependent on receiver support

Reply:
it was discussed in one if the standups last week that once we were confident with the E2E tests, this would be marked alpha
https://docs.google.com/document/d/1qC471RG2YTKw3PjDaqtQ6Rt9U8UaR3yeFwkqV0ZuplA/edit#heading=h.cz8mbgjlmnay

Reply:
Sure, but we shouldn't use `https://github.com/open-telemetry/opentelemetry-collector#alpha` as reference. Until we don't have stability concept in the repository I would either add it to all components or make it descriptive way (like we did with experimental)

Reply:
ok got it, I'll just make it descriptive then like we did with experimental.

Reply:
done